src/pymor/reductors/h2.py

```diff
@@ -66,16 +67,16 @@ def _check_rom0_params(self, rom0_params):
             assert ('sigma', 'b', 'c') in rom0_params
             assert isinstance(rom0_params['sigma'], np.ndarray)
             assert rom0_params['sigma'].ndim == 1
-            assert rom0_params['b'] in self.fom.input_space
-            assert rom0_params['c'] in self.fom.output_space
-            assert len(rom0_params['sigma']) == len(rom0_params['b'])
-            assert len(rom0_params['sigma']) == len(rom0_params['c'])
+            assert rom0_params['b'].shape[1] == self.fom.dim_input
+            assert rom0_params['c'].shape[1] == self.fom.dim_output
+            assert len(rom0_params['sigma']) == rom0_params['b'].shape[0]
+            assert len(rom0_params['sigma']) == rom0_params['c'].shape[0]
```

Comment:
How about adding `assert isinstance(rom0_params['b'], np.ndarray)` above? (And similar for `'c'`.)